static/js/common.ts

```diff
@@ -82,19 +84,29 @@ export function adjust_mac_shortcuts(key_elem_class, require_cmd_style) {
 
 // See https://zulip.readthedocs.io/en/latest/development/authentication.html#password-form-implementation
 // for design details on this feature.
-function set_password_toggle_label(password_selector, label, tippy_tooltips) {
+function set_password_toggle_label(
+    password_selector: string,
+    label: string,
+    tippy_tooltips: boolean,
+): void {
     $(password_selector).attr("aria-label", label);
     if (tippy_tooltips) {
-        if (!$(password_selector)[0]._tippy) {
+        const $el: ReferenceElement<HTMLElement> = $(password_selector)[0];
```

Comment:
This isn’t a jQuery object, so call it `element`, not `$el`.

The generic argument of `ReferenceElement` is the type of `element._tippy.props`, which doesn’t seem to be anything like an `HTMLElement`.

Reply:
Changed the variable name and updated it to just use `ReferenceElement`.